e2e/atlas/alert_settings_test.go

```diff
@@ -157,6 +157,7 @@ func TestAlertConfig(t *testing.T) {
 			"REPLICA_SET_NAME",
 			"SHARD_NAME",
 			"CLUSTER_NAME",
+			"APPLICATION_ID",
```

Comment:
Context, is this a new field that's coming from Atlas or related to any of the work we've been doing?

Reply:
Yes, coming from atlas, I linked the PR in the Jira ticket if you want to take a look!